com.microsoft.mrtk.graphicstools.unity/Samples~/Experimental/Magnifier/Materials.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Looks like some of the materials in this folder aren't used or are duplicates? Can you remove the unused ones?